### DISCUSSION TITLE
Export configuration

### DISCUSSION THREAD

**Discussion by ezawadzki** [engram-design/FeedMe#1110] [Ideas]
Hi,

As we use multiple instances of craft (preprod and prod), it would be very useful to have the possibility to export FeedMe configuration from one instance to be use in another instance.

Do you plan this kind of features ?

Emilie

**Comment by gbowne-quickbase** [comment]
Or at least, since the config CAN be pulled in, if we could export an existing config in the format we'd need to have, then we can modify it based on different instances, and push up changes to feeds as new fields are added, etc. Or easily set the feeds to use different URLs based on the environment
